.travis.yml

```diff
@@ -1,48 +1,25 @@
-language: go
-go:
-  - 1.5.1
-env:
-  - GO15VENDOREXPERIMENT=1
+language: generic
 branches:
   only:
     - master
+env:
+  DEV_REGISTRY=quay.io
```

Comment:
Does this need to be set for CI? The [`deploy.sh` script](https://github.com/deis/workflow-cli/blob/master/_scripts/deploy.sh#L10-L12) sets its own values for `DEV_REGISTRY` as needed.